packages/adapter-netlify/src/serverless.js

```diff
@@ -1,6 +1,7 @@
 import './shims';
 import { Server } from '0SERVER';
 import { split_headers } from './headers.js';
+import { createReadable } from '@sveltejs/kit/node';
```

Comment:
Either the adapters all need major version bumps or you do `import * as node_helpers` so that you can access it defensively below